Cpix.bs.md

```diff
@@ -516,7 +516,7 @@ The key this element contains can be encrypted. If it is encrypted, it is encryp
 :: This is an attribute inherited from [[!RFC6030]] and made optional in this specification.
 
 : <dfn>kid</dfn> (M, xs:string)
-:: The unique identifier of the [=Content Key=].
+:: The unique identifier of the [=Content Key=]. It shall be formatted as defined in [[!MPEGCENC]], section 11.2. In addition, according to [[!MPEGCMAF]], Section 8.2.1, it is strongly recommended the value be a UUID. In this case, it shall be stored sequenced as specified in [[!ITU-X667]], section 6.4.
```

Comment:
The referenced section 6.4 is incomprehensible to me and I thought I know how KIDs work. I don't think this is a good reference to have.

We might be best off avoiding talking about the serialization mechanism in CPIX and just reference the format (given that CPIX only directly references KIDs in string form).

Reply:
This section 6.4 is about UUID in hex format. This is the only existing one and complement what is referenced in CMAF.
Anyway, there is now a bug open on CENC for dealing all this. The last two sentences can be removed. We just reference CENC for the formatting. Make sense ?

Reply:
Sounds good. Can you reference the CENC bug - is it something public on GitHub?

Reply:
Change done. The bug is not public. It is on the MPEG GitLab.